src/liballoc/collections/linked_list.rs

```diff
@@ -916,9 +942,10 @@ impl<T> IterMut<'_, T> {
                issue = "27794")]
     pub fn insert_next(&mut self, element: T) {
         match self.head {
-            None => self.list.push_back(element),
-            Some(mut head) => unsafe {
-                let mut prev = match head.as_ref().prev {
+            None => self.list.push_back(element), // push_back is okay with aliasing nodes
+            Some(head) => unsafe {
+                let prev = match head.as_ref().prev {
+                    // push_back is okay with aliasing nodes
```

Comment:
should that be "push_front is okay with aliasing nodes" ?

Reply:
Indeed, thanks! Fixed.